src/traces/sunburst/plot.js

```diff
@@ -150,6 +150,14 @@ function plotOne(gd, cd, element, transitionOpts) {
     // filter out slices that won't show up on graph
     sliceData = sliceData.filter(function(pt) { return pt.y1 <= cutoff; });
 
+    if(trace.rotation && trace.rotation !== 'auto') {
```

Comment:
There is no `auto` rotation option for this feature.
This line should be 
```js
if(trace.rotation) {
```

Reply:
@archmoj it's not needed here, but unless we make a variant of `valType: 'angle'` that prohibits it, `'auto'` is going to be allowed https://github.com/plotly/plotly.js/pull/5171#discussion_r497545555

Reply:
I'm not sure how to handle this situation. One suggest to include _trace.rotation !== 'auto'_, while an other asks to leave it out.

If you would allow me to vote, I go for leaving the auto-check out.

Once sorted out, I would like to do some more work on the treemap, since noticing different behavior than the sunburst.

Reply:
@thierryVergult I positively reacted to @alexcjohnson's comment. So please keep `trace.rotation !== 'auto'`.
Thanks.

Reply:
didn't saw that. Commit sent.